docs/config.rst

```diff
@@ -1,11 +1,616 @@
-Configuring CumulusCI
+Configure CumulusCI
 =====================
 
-cumulusci.yml
--------------
+The ``cumulusci.yml`` file is located in the ``project`` root directory. This is where you define project dependencies, configure new tasks and flows, customize standard tasks and flows for your project, and so much more! 
```

Comment:
``project`` shouldn't be in backticks here; it's not the literal name of a directory.